config/upload-api.yml

```diff
@@ -167,6 +167,55 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+  /v1/area/{upload_area_id}/owner_emails:
+
+    post:
+      summary: Update owners of an upload area
+      operationId: upload.lambdas.api_server.v1.area.update_area_owners
```

Comment:
nit around naming/description. Based on `test_update_area_owners` it seems like you are replacing the area_owners with whatever data is passed in (as opposed to just adding any new emails). Could you call the function set_area_owners? And specify in the description that it will replace whatever owners are currently attached?